### DISCUSSION TITLE
Improve ease of resolving conflicts between multiple developers changing the schema

### DISCUSSION THREAD

**Discussion by u12206050** [directus/directus#16147] [Feature Requests]
When changes to the schema are made, the single schema file and git sometimes makes it hard to merge changes, forcing us to revert our own changes, apply the other developers changes and then make our changes again before creating a new snapshot and committing it.

### To Reproduce

Have two developers make changes to collections and then create and commit a snapshot
`directus schema snapshot`+ `git commit`

Now have the one that got the conflict message try to merge those conflicts.

**Comment by JulianBissekkou** [comment]
Maybe splitting up the schema into multiple files would be easier for merging. We have the same problem and right now we are able to merge the schema.yaml file but i am sure this won't scale.

**Reply by u12206050** [reply]
Jip, actually have a PR ready with this #16102  But sounds like the team might be working on a similar solution

**Comment by feitzi** [comment]
Does anyone found an acceptable solution for that problem? The PR  #16102 was closed without merging.
Is there any workaround to deal with schema merging between developers?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
